kymatio/scattering1d/utils.py

```diff
@@ -159,8 +122,10 @@ def precompute_size_scattering(J, Q, max_order=2, detail=False):
         integer. If `True`, returns a tuple of size `max_order` containing
         the number of coefficients in each order.
     """
-    sigma_low, xi1, sigma1, j1, xi2, sigma2, j2 = \
-        calibrate_scattering_filters(J, Q)
+    sigma0 = 0.1
```

Comment:
This is repeated. Put a comment or an issue.

Reply:
addressed in 84c77d8